core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -496,12 +498,16 @@ var ExplorationEditorMainTab = function() {
     // Wait for browser to time out the popover, which is 4000 ms.
     await waitFor.invisibilityOf(
       postTutorialPopover, 'Post-tutorial popover does not disappear.');
-
     await waitFor.elementToBeClickable(
       stateEditContent,
       'stateEditContent taking too long to appear to set content');
-    await stateEditContent.click();
+    await waitFor.elementToBeClickable(
+      stateEditButton,
+      'State edit button taking too long to show up');
+    await stateEditButton.click();
```

Comment:
You can use the actions file too 😄 ! You might have to merge from `develop` first to get it.

https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/action.js

Reply:
I've added an `action.click()` there. Thanks for the suggestion!